src/main/cli.py

```diff
@@ -12,7 +12,7 @@
 
 IS_EXIT_1 = False
 COMMIT_EDITMSG = ".git/COMMIT_EDITMSG"
-MAX_MSG_LENGTH = 71
+MAX_MSG_LENGTH = 72
```

Comment:
```
 check_length(msg)
 check_subject_line(msg)
 check_body(msg)
```

Instead of that you can do  if not valid_subject_line and not valid_body then exit

Reply:
`set_args()` why not using proper python params and pass them netween functions instead of global args

Reply:
`  if len(args) < 2:` what is that for?

Reply:
`def set_args():` explain that method